finality-aleph/src/sync/substrate/mod.rs

```diff
@@ -68,11 +68,36 @@ impl<H: SubstrateHeader<Number = BlockNumber>> Header for H {
     }
 }
 
+/// Proper `AlephJustification` or a variant indicating virtual justification
+/// for the genesis block, which is the only block that can be the top finalized
+/// block with no proper justification.
+#[derive(Clone, Debug, Encode, Decode)]
+pub enum InnerJustification {
```

Comment:
This is smth that is only for network usage, right? It will not go into the db?

Reply:
Nope, the stuff going into the DB is what is in this context `AlephJustification` (plus I think a small constant stirng, because substrate?).